styles/variables-silicon.less

```diff
@@ -89,6 +89,14 @@
 // ---------------------------------------
 @agate-imageitem-border-width: 6px;
 
+// Indicator
+// ---------------------------------------
+@agate-indicator-align-self:    center;
+@agate-indicator-border-radius: @agate-indicator-height / 2;
+@agate-indicator-display:       flex;
```

Comment:
I don't think this is a good practice. This has layout structural implications and should be coded into the component file itself. `display:none` removes the component from the layout, rather than just hiding it like `visibility:hidden` or `opacity:0`. Let's find another approach that has a similar visual effect, but doesn't do this.